ardere/step_functions.py

```diff
@@ -403,8 +348,6 @@ def check_for_cluster_done(self):
     def cleanup_cluster(self):
         """Shutdown all ECS services and deregister all task definitions
 
-        Step 7
-
         """
```

Comment:
If you're killing steps, might want to also kill https://github.com/loads/ardere/blob/a7c985b081bc2813148ca4425e994ebddccabca9/ardere/step_functions.py#L369